### PR TITLE
{tools}[GCCcore/10.2.0] x13as v1-1-b59

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/x/x13as/x13as-1-1-b59-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+build_cmd = 'make -f makefile.gf'
+
+files_to_copy = [(["x13as"], "bin")]
+
+sanity_check_paths = {
+    'files': ['bin/x13as'],
+    'dirs': [],
+}
+
+moduleclass = 'tools'
```
Comment (verdurin) [inline easybuild/easyconfigs/x/x13as/x13as-1-1-b59-GCCcore-10.2.0.eb:51]: Should this be `math`?

Comment (sassy-crick) [reply easybuild/easyconfigs/x/x13as/x13as-1-1-b59-GCCcore-10.2.0.eb]: Changed. I am sometimes not really sure if that is a `tool`which is using `math` or just plain `math` being used as a `tool`
:-)
I have also updated PR #16163

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3102.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/2fa5de259065275801718498e10063c3 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**